client/ayon_core/hosts/substancepainter/plugins/load/load_mesh.py

```diff
@@ -25,26 +25,35 @@ class SubstanceLoadProjectMesh(load.LoaderPlugin):
     icon = "code-fork"
     color = "orange"
 
-    options = [
-        qargparse.Boolean(
-            "preserve_strokes",
-            default=True,
-            help="Preserve strokes positions on mesh.\n"
-                 "(only relevant when loading into existing project)"
-        ),
-        qargparse.Boolean(
-            "import_cameras",
-            default=True,
-            help="Import cameras from the mesh file."
-        )
-    ]
-
-    def load(self, context, name, namespace, data):
+    @classmethod
+    def get_options(cls, contexts):
+        return [
+            BoolDef("preserve_strokes",
+                    default=True,
+                    label="Preserve Strokes",
+                    tooltip=("Preserve strokes positions on mesh.\n"
+                             "(only relevant when loading into "
+                             "existing project)")),
+            BoolDef("import_cameras",
+                    default=True,
+                    label="Import Cameras",
+                    tooltip="Import cameras from the mesh file."
+            ),
+            EnumDef("texture_resolution",
+                    items=[128, 256, 512, 1024, 2048, 4096],
+                    default=1024,
+                    label="Texture Resolution",
+                    tooltip="Set texture resolution for the project")
```

Comment:
Should the tooltip elaborate that this only works for "NEW" projects - not for loading into an existing project? Or maybe we put in a "divider" in the definitions with a label like "New Project Settings:" or alike?